src/main/generated/resources/data/minestuck/recipes/grist_costs/spork.json

```diff
@@ -4,7 +4,6 @@
     "item": "minestuck:spork"
   },
   "grist_cost": {
-    "minestuck:build": 50,
-    "minestuck:rust": 9
+    "minestuck:build": 13
   }
 }
```

Comment:
Earlier I said I didnt think this was a problem. Searching through the spreadsheet where we handle grist costs I have realized it is a problem, it should be returned back to the deleted costs by going to MinestuckGristCostsProvider and manually setting it again